lib/requests.js

```diff
@@ -174,7 +174,7 @@ function _sanitizeHeaders({httpMessage, headers = sanitizeHeaders}) {
   for(const header of headers) {
     // sanitize the headers to prevent
     // authn tokens / information potentially in logs
-    newHeaders.set(header, '** SANITIZED TO PREVENT EXPOSING OF SECRETS ***');
+    newHeaders.set(header, 'sanitized');
```

Comment:
I agree there's no need to SHOUT. But I think the more informational text is appropriate.
```suggestion
    newHeaders.set(header, 'sanitized to prevent exposure of secrets');
```